src/modules/robot/Robot.cpp

```diff
@@ -396,22 +396,20 @@ void Robot::on_gcode_received(void *argument)
                     this->max_speeds[Y_AXIS] = gcode->get_value('Y');
                 if (gcode->has_letter('Z'))
                     this->max_speeds[Z_AXIS] = gcode->get_value('Z');
-                if (gcode->has_letter('A'))
-                    alpha_stepper_motor->set_max_rate(gcode->get_value('A'));
-                if (gcode->has_letter('B'))
-                    beta_stepper_motor->set_max_rate(gcode->get_value('B'));
-                if (gcode->has_letter('C'))
-                    gamma_stepper_motor->set_max_rate(gcode->get_value('C'));
-
+                for (size_t i = 0; i < actuators.size(); i++) {
+                    if (gcode->has_letter('A' + i))
```

Comment:
It makes little sense keeping up with this format as we add more actuators/axes.
I'd much rather we didn't modify this gcode to accomodate more axes/actuators, and made a *new* gcode that doesn't use letters, but rather answers something like 

axes:0,10,0,0,5,0 actuators:10,30,10,5,0,5